shared/src/business/entities/cases/Case.js

```diff
@@ -1741,6 +1745,18 @@ Case.prototype.removeConsolidation = function () {
   return this;
 };
 
+/**
+ * checks all the practitioners on the case to see if there is a privatePractitioner associated with the userId
+ *
+ * @param {String} userId the id of the user
+ * @returns {boolean} if the userId has a privatePractitioner associated with them
+ */
+Case.prototype.isUserIdRepresentedByPrivatePractitioner = function (userId) {
```

Comment:
Just to keep coverage up and intent clear, we should probably have some tests around the new functions on the `Case` entity and `DocketEntry` too.

Reply:
Good catch, I made a mental note to go back and check that, but got wrapped up in this migration.